### ISSUE TITLE
deprecate/remove/upgrade PyRTF?

### ISSUE THREAD

**Issue by Anthchirp** [cctbx/cctbx_project#108]
We currently have a dependency on PyRTF in the bootstrapper. PyRTF is not Python3 compatible.
PyRTF looks to be [pretty dead](http://pyrtf.sourceforge.net/). There is a fork which has been [dead for 8+ years, too](https://github.com/nekstrom/pyrtf-ng). There is also [PyRTF3](https://pypi.python.org/pypi/PyRTF3) which I guess aims to become a Python3 compatible version of PyRTF, but its documentation does not give me the impression that it is stable yet.

As far as I can tell PyRTF is only used in 2 places: [iotbx/table_one.py](https://github.com/cctbx/cctbx_project/blob/master/iotbx/table_one.py) (related lines are all >4 years old) and in phenix in automation/structure_comparison/export.py (last update 5+ years ago).

So I guess the first question should be *"do we want to keep this?"*, and then we can decide what to do about it. Discuss.

**Comment by bkpoon** [comment]
RTF output is important because that's an easy way for users to copy Table 1 statistics into their manuscripts. I wouldn't expect there to be many updates since it's just exporting text into a mature format. We don't print anything complicated, so I would actually be concerned if we needed to continually update code to convert text to RTF.

PyRTF3 looks like a possible upgrade except for the GPL licensing from the pypi website. However, the GitHub project for PyRTF3 is under a MIT license. I suspect the PyRTF3 maintainer didn't update the license file.

But I did a quick conversion of PyRTF (from https://launchpad.net/pyrtf, apparently the code moved from Sourceforge to Google Code and finally to Launchpad) to Python 3 and the tests in the package pass, so that could be used instead of the PyRTF3.
